### ISSUE TITLE
E1125 (missing-kwoa) triggers when passing a keywords dict

### ISSUE THREAD

**Issue by jix** [PyCQA/pylint#934]
### Steps to reproduce

The following (python 3 only) example produces an inconsistency in handling of positional and keyword arguments:

```python
def target(pos, *, keyword):
    print(pos, keyword)


def forwarding_kwds(pos, **kwds):
    target(pos, **kwds)


def forwarding_args(*args, keyword):
    target(*args, keyword=keyword)
```

### Current behavior

The call `taget(pos, **kwds)` produces a `Missing mandatory keyword argument 'keyword' in function call (missing-kwoa)` error lint, while the analogous case for positional arguments `target(*args, keyword=keyword)` does not.

### Expected behavior

I would expect both cases to be handled the same. As forwarding of arguments is commonly done, I'd expect both to not trigger an error lint.

### pylint --version output

```
% pylint --version
No config file found, using default configuration
pylint 1.5.4, 
astroid 1.4.5
Python 3.5.1 (default, Mar  3 2016, 09:29:07) 
[GCC 5.3.0]
```

**Comment by PCManticore** [comment]
Thanks for reporting this! It might be easier to fix, so I'm tentatively adding it for the next milestone.
